input/postgres/relations.go

```diff
@@ -36,7 +36,9 @@ const relationsSQL string = `
 				(SELECT p.partattrs FROM pg_partitioned_table p WHERE p.partrelid = c.oid) AS partition_columns,
 				COALESCE(pg_catalog.pg_get_partkeydef(c.oid), '') AS partition_expr,
 				locked_relids.relid IS NOT NULL AS exclusively_locked,
-				COALESCE(toast.relname, '') AS toast_table
+				COALESCE(toast.relname, '') AS toast_table,
+				coalesce(pg_relation_filenode(c.oid), 0) AS data_filenode,
+				coalesce(pg_relation_filenode(c.reltoastrelid), 0) AS toast_filenode
```

Comment:
```suggestion
				COALESCE(toast.relname, '') AS toast_table,
				COALESCE(pg_relation_filenode(c.oid), 0) AS data_filenode,
				COALESCE(pg_relation_filenode(c.reltoastrelid), 0) AS toast_filenode
```
Or the other way around (assuming you change other occurrences here). I don't especially care, but the inconsistency hurts readability.